pkgs/top-level/php-packages.nix

```diff
@@ -540,7 +540,12 @@ lib.makeScope pkgs.newScope (self: with self; {
             ++ lib.optionals (lib.versionOlder php.version "7.4") [ "--with-libxml-dir=${libxml2.dev}" ];
           doCheck = false;
         }
-        { name = "sockets"; doCheck = false; }
+        {
+          name = "sockets";
+          doCheck = false;
+          patches = lib.optional (lib.versionAtLeast php.version "8.1.2")
+            ../development/interpreters/php/7981.patch;
```

Comment:
can't we just `fetchpatch` this?

Reply:
Oooh never used that yet! Going to update the patch.

Reply:
Excellent, thanks for the suggestion, it works !